src/hooks/useUpdateNickname.ts

```diff
@@ -0,0 +1,25 @@
+import { useAtom } from 'jotai';
+
+import { userAtom } from '@stores';
+
+type nickNameType = 'hostNickname' | 'guestNickname';
+
+export const useUpdateNickname = () => {
+  const [user, setUser] = useAtom(userAtom);
+  const updateNickname = (nickname: nickNameType, updateNickname: string) => {
+    if (nickname === 'hostNickname') {
+      setUser({ ...user, hostNickname: updateNickname });
+    } else if (nickname === 'guestNickname') {
+      setUser({ ...user, guestNickname: updateNickname });
+    }
```

Comment:
```suggestion
  const updateNickname = (nickname: nickNameType, updateNickname: string) => {
    nickname === 'hostNickname'  ? setUser({ ...user, hostNickname: updateNickname });
     : setUser({ ...user, guestNickname: updateNickname });
    }
```
기왕 간지나게 작성할거면 이렇게 삼항연산자로 해도 될 것 같아요!
본인의 선택임다 ㅎㅎ

Reply:
오 이거 진짜 간지나네요,, 이걸로 수정하겠습니다!! 감사합니다 ㅎㅎ

Reply:
코드를 줄이고 싶은거면 

```
setUser({ ...user, [nickname]: updateNickname });
```

이런식으로 가능하지않나요?

Reply:
세상에 그러네요 !!! 최고다 최고,, 반영하겠습니다!! 🥹